src/swagger/get.ts

```diff
@@ -72,11 +74,10 @@ export const getSwagger = (
     host = SwaggerDefaultConfig.HOST,
     schemes = [SchemeType.Http],
     auth,
-    authOptions,
     ...otherOptions
   } = options;
 
-  const authentication = getAuthentication(auth, authOptions);
+  const authentication = getAuthentication(auth);
```

Comment:
Here is where we actually get the root-level security details for the Swagger docs.